README.md

```diff
@@ -46,15 +46,15 @@ Awesome! Now, you'll want to configure a client.
 You'll need to configure *one* client for *each* OAuth2 server
 (GitHub, Facebook, etc) that you want to talk to.
 
-### Step 1) Download the client library
+### Step 1) Download the Client Library
 
 Choose the one you want from this list and install it
 via Composer:
 
 <a name="client-downloader-table"></a>
 
 | OAuth2 Provider                                                        | Install                                             |
-| ---------------------------------------------------------------------- | ------------------------------------------------------- |
+| ---------------------------------------------------------------------- | --------------------------------------------------- |
```

Comment:
If this is too long, I think it needs to be fixed in `bin/update_readme`

Reply:
Ah, really! Now with #362 we won't forget about it :)

Reply:
The actual change is done in #360